lib/kontena/plugin/vagrant/master/create_command.rb

```diff
@@ -23,6 +25,18 @@ def execute
       )
     end
 
+    def ask_instance_memory
```

Comment:
We need a bit more control when these are called.. I think if we use clamp magic we cannot control that? Or are these triggered only when `self.foo` is called?